### PR TITLE
fix checkpoint path and add timeseries publisher integration test

### PR DESCRIPTION
**Pull request by seblapela**
## References

* #463

### CODE REVIEW

File: source/databricks/streaming-jobs/timeseries_publisher_streaming.py [hunk 1]
```diff
@@ -23,18 +23,16 @@
...
 args, unknown_args = p.parse_known_args()
-
 spark = initialize_spark(args)
 
-timeseries_unprocessed_path = f'abfss://{args.delta_lake_container_name}@{args.data_storage_account_name}.dfs.core.windows.net/{args.timeseries_unprocessed_blob_name}'
-time_series_points_path = f'abfss://{args.delta_lake_container_name}@{args.data_storage_account_name}.dfs.core.windows.net/{args.time_series_points_blob_name}'
-checkpoint_path = f"abfss://{args.delta_lake_container_name}@{args.storage_account_name}.dfs.core.windows.net/checkpoint-timeseries-publisher"
+timeseries_unprocessed_path = f"{args.timeseries_unprocessed_path}"
+time_series_points_path = f"{args.time_series_points_path}"
+checkpoint_path = f"{args.checkpoint_path}"
```
Comment (BjarkeMeier) [inline source/databricks/streaming-jobs/timeseries_publisher_streaming.py:35]: nit: Consider consisten naming here to.

File: source/databricks/tests/integration/time_series_publisher/test_time_series_publisher.py [hunk 2]
```diff
@@ -26,6 +27,25 @@
...
+        f"{databricks_path}/streaming-jobs/timeseries_publisher_streaming.py",
+        "--data-storage-account-name", "data-storage-account-name",
+        "--data-storage-account-key", "data-storage-account-key",
+        "--timeseries_unprocessed_path", f"{delta_lake_path}/unprocessed_time_series",
+        "--time_series_points_path", f"{delta_lake_path}/time_series_points",
+        "--checkpoint_path", f"{delta_lake_path}/time_series_points/checkpoint",
+        ])
+
+    # Assert
+    assert exit_code == 0, "Preparation job did not return exit code 0"
```
Comment (BjarkeMeier) [inline source/databricks/tests/integration/time_series_publisher/test_time_series_publisher.py:46]:
```suggestion
    assert exit_code == 0, "Time-series publisher job did not return exit code 0"
```

File: source/databricks/tests/integration/time_series_publisher/test_time_series_publisher.py [hunk 3]
```diff
@@ -44,7 +64,7 @@ def time_series_publisher(spark, delta_lake_path, integration_tests_path, unproc
     time_series_data = [(unprocessed_time_series_json_string, 2022, 3, 21, "2022-12-17T09:30:47Z")]
     (spark
      .sparkContext
-     .parallelize(time_series_data)
+     .parallelize(time_series_data)
```
Comment (BjarkeMeier) [inline source/databricks/tests/integration/time_series_publisher/test_time_series_publisher.py:67]: Whitespaces added - probably unintentionally.

**Review by BjarkeMeier — APPROVED** [review]
LGTH - only nit comments.

**Comment by seblapela** [comment]
@cla-bot check

**Comment by seblapela** [comment]
@cla-bot check

**Comment by seblapela** [comment]
@cla-bot check

**Pull request merged**